apps/mark-scan/frontend/src/app_root.tsx

```diff
@@ -545,6 +561,9 @@ export function AppRoot({ reload }: Props): JSX.Element | null {
       }
     }
 
+    // Implicitly map resetting_state_machine_no_delay to return InsertCardScreen
+    // because that state implies we are returning to the starting (default) state.
```

Comment:
Could this be made explicit (e.g. with an `assert`) or by masking `resetting_state_machine_no_delay` on the backend by returning a state that would go to the insert card screen?

Reply:
[80b0193](https://github.com/votingworks/vxsuite/pull/4647/commits/80b0193548f15f61b0685553efcd675edfe4155b)